platform/admin/ui/src/main/webapp/templates/installer/finish.handlebars

```diff
@@ -22,7 +22,7 @@
             to change hostname, ports and certificates.
             <br/>
             <br/>
```

Comment:
:question: Should ${home_directory} be used instead of INSTALL_HOME?

Reply:
That's a good question. I am open to it. I would hope users wouldn't get confused by that and look in their home directory rather than where they "unzipped" DDF.